include/envoy/runtime/runtime.h

```diff
@@ -30,10 +31,30 @@ class RandomGenerator {
 public:
   virtual ~RandomGenerator() = default;
 
+  using result_type = uint64_t;
```

Comment:
Nope, but it is a part of `UniformRandomBitGenerator`[1] interface.  If it looks ugly/weird inside this class, I can move it to a separate "adaptor" class.

[1] https://en.cppreference.com/w/cpp/named_req/UniformRandomBitGenerator

Reply:
hmmm clang-tidy borked but "passed" the PR cc @lizan 
```
...
Running clang-tidy-diff against master branch...
From https://github.com/envoyproxy/envoy
 * branch            master     -> FETCH_HEAD
fatal: ambiguous argument 'master..HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
No relevant changes found.
```

(it should be flagging this line because codebase convention is for [type aliases to be CamelCase](https://github.com/envoyproxy/envoy/blob/master/.clang-tidy#L84), but can be overridden with `// NOLINT(readability-naming-identifier)` for cases like this when conforming to a library [such as here](https://github.com/envoyproxy/envoy/blob/e2cf41606c42cd18b5ac5fd2add9cf983ece9172/source/common/stats/allocator_impl.h#L39))

Reply:
opened https://github.com/envoyproxy/envoy/pull/9140 to fix clang-tidy

Reply:
Added lint comment.